packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -206,16 +234,15 @@ export const GiantTrackTile = ({
 }: GiantTrackTileProps) => {
   const dispatch = useDispatch()
   const [artworkLoading, setArtworkLoading] = useState(false)
+  const { isEnabled: isSearchV2Enabled } = useFeatureFlag(
+    FeatureFlags.SEARCH_V2
```

Comment:
Does this flag already exist in Optimizely? If not, mind creating it _before_ you merge so we don't get a bunch of sentry errors about it?

Reply:
it exists!